static/styles/rewards/claim-table.css

```diff
@@ -168,8 +170,15 @@ table[data-make-claim-rendered] tr#additional-details-border + tr > * {
 table #additionalDetailsTable {
   opacity: 0;
   pointer-events: none;
-  transform: translate(-50%, -90px);
+  transform: translate(-50%, -133px);
 }
+
+#table-target table[data-details-visible="true"]:nth-child(even) #additionalDetailsTable {
+  opacity: 1;
+  pointer-events: none;
+  transform: translate(-50%, -70px);
```

Comment:
This works for more than two tables?

Reply:
Yes it does work for both

Reply:
For more than two tables. Like three or more.

Reply:
You're right works fine for 3 but upwards of that it breaks. I'm looking into it thanks